tests/__init__.py

```diff
@@ -15,10 +15,17 @@ def check_output(*args, **kwargs):
     stdout = stdout.replace(b"\r\n", b"\n")
     return stdout
 
-def cmake(cwd, targets, options=[]):
+def cmake(cwd, targets, options=None):
+    if options is None:
+        options = {}
+    options.update({
+        "CMAKE_RUNTIME_OUTPUT_DIRECTORY": cwd,
```

Comment:
👍 this indeed looks like the better place for this than inside the cmake file!

Reply:
The only disadvantage is that a quick `cmake .. && make` will create the test executable in the `test/unit` subfolder.
But you can always run `make test`.